helper/config-json-parser.go

```diff
@@ -134,6 +136,20 @@ func GetAccessToken(file []byte) string {
 	return config.AccessToken
 }
 
+func GetRefreshToken(file []byte) string {
```

Comment:
This function seems to duplicate the exact code as `GetAccessToken`. Can we not reuse the same function for getting refresh token?
May be use a flag to determine if you need access-token/refresh-token.

Reply:
As technically the access token and refresh token are different, hence created two different functions. And also if I merge both in one function, I have to change the name of the function, which will again require end to end testing of plugin.